<?xml version="1.0" encoding="utf-8" ?>
<launch>
    <arg name="hdf5FolderPath" default="/seerep/seerep-data/cognitiveWeeding/fr/"/>
    <arg name="jsonFilePath" default="/seerep/seerep-data/cognitiveWeeding/fr/data.json"/>
    <arg name="classesMappingPath" default="/seerep/seerep-data/cognitiveWeeding/fr/label_seerep_fr.json"/>
    <arg name="dataSource" default="fr"/> <!-- fr or uos -->

    <node name="seerep_ros_comm_point_dumper" pkg="seerep_ros_comm" type="point_dumper" output="screen">
        <rosparam param="hdf5FolderPath" subst_value="True">$(arg hdf5FolderPath)</rosparam>
        <rosparam param="jsonFilePath" subst_value="True">$(arg jsonFilePath)</rosparam>
        <rosparam param="classesMappingPath" subst_value="True">$(arg classesMappingPath)</rosparam>
        <rosparam param="dataSource" subst_value="True">$(arg dataSource)</rosparam>
    </node>
</launch>
